Fixed: organisation tree in humanres doesn't show person names (OFBIZ-11677)

PierreSmits opened the pull request:
(OFBIZ-11677)

When parties are assigned to employee positions their names aren't shown in the tree
fixed: the function that retrieves the person record of the assigned party was using
	the wrong variable in the query, resulting in a null being returned. Corrected the variable
added/improved: retrieval of the middleName value of the person record, added it to the title
improved: ordering of the names making up the title.
improved: applying proper case to variable names

PierreSmits committed:
Fixed: Tree on main page of the humanres component doesn't show names of persons

(OFBIZ-11677)

When parties are assigned to employee positions their names aren't shown in the tree
fixed: the function that retrieves the person record of the assigned party was using
	the wrong variable in the query, resulting in a null being returned. Corrected the variable
added/improved: retrieval of the middleName value of the person record, added it to the title
improved: ordering of the names making up the title.
improved: applying proper case to variable names

pwverma commented:
Hello @PierreSmits 

I would use PartyHelper.getPartyName() instead of fetch person and party group record(just below) to get the name. WDYT?

PierreSmits commented:
@verma-pawan That is a good idea. Thanks for the suggestion. I will look into it.

PierreSmits committed:
Fixed: Tree on main page of the humanres component doesn't show names of persons

(OFBIZ-11677)

invoking PartyHelper.getPartyName instead of custom code
code clean-up

Thanks to Pawan Verma for the review and the suggestion

PierreSmits commented:
@verma-pawan Your suggestion has been implemented, tested and committed into the PR.

pwverma commented:
Thanks, @PierreSmits 
It contains some extra changes about code improvement, it looks good to me.

PierreSmits commented:
Never waste a moment. 😄

PierreSmits commented:
@verma-pawan 

I trust that nothing now stands in the way to get this merged into the code base. Will you assist?

PierreSmits commented:
Anyone willing to assist (review) to get this PR merged into the code base?

JacquesLeRoux commented:
I'll try this afternoon...

PierreSmits commented:
Thanks @JacquesLeRoux

JacquesLeRoux commented:
Thanks Pawan
